src/subsystems/php/builders/granular-php/default.nix

```diff
@@ -36,26 +37,30 @@
 
     inherit (pkgs.callPackage ../../semver.nix {}) satisfies;
 
-    # php with required extensions
-    php =
-      if satisfies pkgs.php81.version subsystemAttrs.phpSemver
+    phpPackages = l.attrValues inputs.phps.packages.x86_64-linux;
```

Comment:
```suggestion
    phpPackages = l.attrValues inputs.phps.packages.x86_64-linux;
    phpPackages = l.attrValues (import inputs.phps).packages.x86_64-linux;
```

Reply:
For my own understanding, what does import do here?

Reply:
Instead of treating `inputs.php` like a flake, it treats it as a path and imports it. Currently we have a failing CI tests, because under some circumstances inputs.php is a path and not a flake. This is buggy behavior in our backward compat mechanism which I don't intend to fix anymore in favor of the upcoming v1 API changes.